examples/minimal_cxx_qt_qml/src/tests/myobject/tst_myobject.qml

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+// SPDX-FileContributor: Gerhard de Clercq <gerhard.declercq@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+import QtQuick 2.12
+import QtTest 1.12
+
+import com.kdab.cxx_qt.demo 1.0
+
+TestCase {
+    name: "MyObjectTests"
+
+    Component {
+        id: componentMyObject
+
+        MyObject {
+
+        }
+    }
+
+    Component {
+        id: componentSpy
+
+        SignalSpy {
+
+        }
+    }
+
+    function test_increment() {
+        const myObject = createTemporaryObject(componentMyObject, null, {});
+        compare(myObject.incrementNumber(1), 2);
```

Comment:
This fails as increment_number on the rust side doesn't return any more and doesn't take a parameter.

So instead use
```qml
myObject.incrementNumber();
compare(myObject.number(), 2);
```

Reply:
Done